azure-pipelines.yml

```diff
@@ -11,9 +11,9 @@ variables:
   _SignType: real
   _TeamName: DotNetSpark
   MSBUILDSINGLELOADCONTEXT: 1
-  # backwardCompatibleRelease/forwardCompatibleRelease is the "oldest" releases that work with the current release
-  backwardCompatibleRelease: '0.9.0'
+  # forwardCompatibleRelease/backwardCompatibleRelease is the "oldest" releases that work with the current release
   forwardCompatibleRelease: '0.9.0'
+  backwardCompatibleRelease: '0.9.0'
```

Comment:
any reason the order got flipped?

Reply:
I just thought flip the order make it more readable. But I can get it back. Please let me know what is preferred.

Reply:
ok, not sure why it's more readable, but it's fine.